recipes/f77_zmq/build.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+./configure --prefix=${PREFIX} --libdir=${PREFIX}/lib --includedir=${PREFIX}/include
+
+make
+FAILURE=0
+
+# Check if we are on Linux
+if [[ -f ${PREFIX}/lib/libzmq.so ]] ; then
+  make check || FAILURE=$?
+  if [[ $FAILURE -ne 0 ]] ; then
+    cat ./test-suite.log
+    exit $FAILURE
+  fi
+fi
+make install
```

Comment:
We don't allow shipping static libs. Don't build them or remove them during the build step.

Reply:
Done